src/styles/view.styl

```diff
@@ -30,6 +30,17 @@
 .set-view-subtitle
     margin-bottom 0
 
+.set-view-content-twoFA-modal
+    &-button
+        text-align center
+.set-view-content-twoFA-point
+    display flex
+    padding-bottom 10px
+    align-items start
+    img
```

Comment:
Could you avoid targetting directly `<img>`? I will suggest `set-view-content-twoFA-image` if it's possible